src/Umbraco.Web.UI.Client/src/common/services/umbdataformatter.service.js

```diff
@@ -243,7 +243,7 @@
                     return item.alias === "_umb_membergroup";
                 });
                 saveModel.email = propEmail.value;
```

Comment:
@RobertCopilau any reason we aren't trimming the email here if we are doing that on the server side and we're trimming login here?

Reply:
I'm adding trim in ... but it seems to not really matter that much since there's a bunch of UI validation actually preventing this from ever being submitted with a trailing space (I guess that's just part of email field standard validation) ... but it doesn't hurt to have it there.